app/assets/javascripts/components/articles/article.jsx

```diff
@@ -11,11 +11,21 @@ const Article = createReactClass({
 
   propTypes: {
     article: PropTypes.object.isRequired,
+    index: PropTypes.number,
     course: PropTypes.object.isRequired,
     fetchArticleDetails: PropTypes.func.isRequired,
     articleDetails: PropTypes.object,
     wikidataLabel: PropTypes.string,
-    showOnMount: PropTypes.bool
+    showOnMount: PropTypes.bool,
+    setSelectedIndex: PropTypes.func,
+    lastIndex: PropTypes.number,
+    selectedIndex: PropTypes.number
+  },
+
+  getInitialState() {
```

Comment:
Does this do anything in Article? I don't see state used at all in this component.